ui/webui/test/README.rst

```diff
@@ -243,3 +266,58 @@ development environment that's independent from the host::
 
     toolbox create --image quay.io/cockpit/tasks -c cockpit
     toolbox enter cockpit
+
+Web UI End to end tests
+=======================
+
+End to end tests are stored in ``./ui/webui/test/integration``
+together with tplib test cases and test plans (those are only needed when running
+the tests using Permian).
+
+End to end tests use one more level of abstraction, class `IntegrationTest <https://github.com/rhinstaller/anaconda/blob/anaconda-39.16-1/ui/webui/test/helpers/integration.py#L38>`_.
+This class handles flow through all the required installation steps with default
+options. So when writing new test you only have to use this class as parent and
+extend it or reimplement the functions that are important for the test case.
+
+End to end tests examples
+--------------------------
+
+There are three test cases in the anaconda repository that can be used as examples.
+
+**Default**
+
+Performs default installation.
+Test script ``default.py``, test case file ``default.tc.yaml``.
+
+**Storage encryption**
+
+Makes changes, compared to default installation, only in the storage section of
+the installation wizard and runs some commands before the system is rebooted.
+Test script ``storage_encryption.py``, test case file ``storage_encryption.tc.yaml``.
+
+**Wizard navigation**
+
+Changes the way how the test steps through the installation wizard.
+Test script ``wizard_navigation.py``, test case file ``wizard_navigation.tc.yam``.
+
+Running End to end tests
+-------------------------
+
+Usually they are run by a Permian workflow and this is the recommended way,
```

Comment:
Suggeston for rephrase to imrpove reability:

The recommended way to run these tests is through a Permian workflow, which is explained in detail in the documentation available [here](https://permian.readthedocs.io/en/devel/workflows/anaconda-webui.html). Alternatively, you can manually set up the environment and execute the tests individually. Please refer to the `Preparation and general invocation`_ section for this.

For a comprehensive execution, including post-reboot checks, the tests need to be executed on an existing machine where the installer is running with the cmdline options 'inst.sshd inst.webui.remote'.